src/components/App/AppIFrame.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import { styled } from '@aragon/ui'
+import { clamp, lerp } from '../../math-utils'
+import { noop } from '../../utils'
+import AppLoadingProgressBar from './AppLoadingProgressBar'
+
+const LOADING_START = 25 // Start loading indicator at 25%
+const LOADING_END = 100
+const LOADING_FUDGE_LIMIT = 75 // Limit arbitrary incremental movement of loading incator to 75%
+
+// See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe for details about sandbox
+// `sandbox` works like a whitelist: by default, almost every functionality is restricted.
+const SANDBOX = [
+  // Allows new browsing contexts (e.g. new windows or tabs) generated by the
+  // iframe to escape the sandbox restrictions. Otherwise, new contexts would
+  // share the same restrictions as their originating iframe (in our case, the
+  // new contexts would only have the ability to run JavaScript).
+  'allow-popups-to-escape-sandbox',
+
+  // Gotta run 'em all!
+  'allow-scripts',
+
+  // Note that we haven't enabled:
+  //   - 'allow-same-origin':
+  //       The most important security setting: leaving this disabled lets the
+  //       iframe be considered as coming from a unique, orphan origin. This
+  //       means that the page won't have access to any cookies, local/session
+  //       storage, or access to open pages (e.g. the parent window, this Dapp).
+  //
+  //       However, this does force some restrictions:
+  //         - `window.postMessage()` must use `*` as an origin to communicate
+  //           with this iframe
+  //         - React devtools can't be hooked in from the browser, so you have
+  //           to use the native `react-devtools` electron app for debugging
+  //
+  //  - 'allow-top-navigation':
+  //       Leaving this disabled disallows the iframe from navigating the
+  //       Dapp's context (e.g. to a malicious page)
+].join(' ')
```

Comment:
I want to somehow audit these permissions to make sure we are good :)

Reply:
I did this when choosing them for my old example wrapper by trying to break out of the iframe, these are fine.

We can always make a malicious app we can test against :)